package.json

```diff
@@ -38,7 +38,9 @@
     "test-all": "run-p flow license-check lint test",
     "test-build-coverage": "jest --coverage --coverageReporters=html",
     "test-serve-coverage": "ws -d coverage/ -p 4343",
-    "test-coverage": "run-s test-build-coverage test-serve-coverage"
+    "test-coverage": "run-s test-build-coverage test-serve-coverage",
+    "test-alex": "yarn alex ./docs-* *.md"
```

Comment:
I think you don't need `yarn` at the start of the command.

Reply:
👍